integration-tests/src/test/java/oracle/weblogic/kubernetes/ItManageNs.java

```diff
@@ -0,0 +1,614 @@
+          false, "", "scaleDown", 1, "", "", null, null);
+      return false;
+
+    } catch (ConditionTimeoutException ex) {
+      logger.info("Received expected error " + ex.getMessage());
+      return true;
+    }
+  }
+
+  private static void setLabelToNamespace(String domainNS, Map<String, String> labels) {
+    //add label to domain namespace
+    V1Namespace namespaceObject1 = assertDoesNotThrow(() -> Kubernetes.getNamespaceAsObject(domainNS));
+    assertNotNull(namespaceObject1, "Can't find namespace with name " + domainNS);
+    namespaceObject1.getMetadata().setLabels(labels);
+    assertDoesNotThrow(() -> Kubernetes.replaceNamespace(namespaceObject1));
+  }
+
+  private void checkOperatorCanScaleDomain(String opNamespace, String domainUid) {
+    int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+    assertTrue(scaleClusterWithRestApi(domainUid, clusterName, 3,
+        externalRestHttpsPort, opNamespace, OPERATOR_RELEASE_NAME + "-sa"),
+        "Domain " + domainUid + " scaling operation failed");
+  }
+
+  private void checkSecondOperatorFailedToShareSameNS(String domainNamespace) {
+    // try to install another operator sharing same domain namespace via different domainNsSelectionStrategy
+    HelmParams opHelmParams = new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespaces[2])
+        .chartDir(OPERATOR_CHART_DIR);
+    try {
+      HelmParams opHelmParams2 = installAndVerifyOperator(OPERATOR_RELEASE_NAME,
+          opNamespaces[2], "List",
+          null, true, domainNamespace);
+      assertNull(opHelmParams2, "Operator helm chart sharing same NS with other operator did not fail");
+    } catch (org.opentest4j.AssertionFailedError ex) {
+      //expecting to fail
+      logger.info("Helm installation failed as expected " + ex.getMessage());
+      uninstallOperator(opHelmParams);
+    }
+  }
+
+  private void checkDomainNotStartedInDefaultNS() {
+    //verify operator can't start domain in the default namespace when domainNsSelectionStrategy not List
+    // and selector does not match default
+    checkPodNotCreated("defaultuid" + adminServerPrefix, "defaultuid", "default");
+
+    logger.info("Delete defaultuid custom resource in namespace {0}", "default");
+    deleteDomainCustomResource("defaultuid", "default");
+    logger.info("Deleted Domain Custom Resource " + "defaultuid");
+  }
+
+  private boolean createDomainCrdAndVerifyDomainIsRunning(String domainNamespace, String domainUid) {
+
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    Domain domain = createDomainCRD(domainNamespace, domainUid);
+    assertDoesNotThrow(() -> createVerifyDomain(domainNamespace, domainUid, miiImage, domain));
+    return true;
+  }
+
+  /**
+   * Create a model in image domain crd.
+   */
+  private Domain createDomainCRD(String domainNamespace, String domainUid) {
```

Comment:
Again here we are creating Domain Resource not CRD. Modify the method name to createDomainResource()

Reply:
done